build.gradle

```diff
@@ -141,12 +139,15 @@ configure(subprojects.findAll { it.name != 'util' }) {
     javadoc {
         exclude "**/internal/**"
 
-        dependsOn project(':util').compileJava //We need taglets to be compiled
+        dependsOn project.project(':util').compileJava //We need taglets to be compiled
```

Comment:
For my edification... what necessitated this change?  What does project.project mean?

Reply:
that's a remnant of some refactoring to the new gradle API.   I'll clean that up in a subsequent PR.  It looks like the docs aren't generating valid links anymore to the mongo docs so i'll have a PR to clean that up soon once I track it down.